src/ElementQueries.js

```diff
@@ -350,25 +351,36 @@
             }
             if ('string' === typeof rules) {
                 rules = rules.toLowerCase();
-                if (-1 !== rules.indexOf('min-width') || -1 !== rules.indexOf('max-width')) {
+                if (isElementQuery(rules)) {
                     extractQuery(rules);
                 }
             } else {
-                for (var i = 0, j = rules.length; i < j; i++) {
-                    if (1 === rules[i].type) {
-                        selector = rules[i].selectorText || rules[i].cssText;
-                        if (-1 !== selector.indexOf('min-height') || -1 !== selector.indexOf('max-height')) {
-                            extractQuery(selector);
-                        }else if(-1 !== selector.indexOf('min-width') || -1 !== selector.indexOf('max-width')) {
-                            extractQuery(selector);
-                        }
-                    } else if (4 === rules[i].type) {
-                        readRules(rules[i].cssRules || rules[i].rules);
+                for (var i = 0, j = rules.length, r; i < j; i++) {
+                    r = rules[i];
+                    switch (r.type) {
+                      case 1:   // CSSRule.STYLE_RULE
+                          selector = r.selectorText || r.cssText;
+                          if (isElementQuery(selector)) {
+                              extractQuery(selector);
+                          }
+                          break;
+                      case 3:   // CSSRule.IMPORT_RULE
+                          r = r.styleSheet;  // then continue to next case statement
```

Comment:
It's hard to understand, that's why I prefer not to have switch-case especially when they contain non breaking cases. It appears to me like it's wrong, because you overwrite `r` in `case 3` because there is only `case 4` that can react on that case. `STYLE_RULE` in `case 1` is not checked for that new `r`.